src/app/components/mnoe-users-list/mno-users-list.coffee

```diff
@@ -93,7 +99,7 @@
 
     displayCurrentState = () ->
       setAllUsersList()
-      fetchUsers(scope.users.nbItems, 0, scope.users.sortAttr)
+      # fetchUsers(scope.users.nbItems, 0, scope.users.sortAttr)
```

Comment:
If this is unnecessary should it simply be remove instead of commented out?

Reply:
Aah yes it was supposed to be removed. Updating